src/components/HarvestersDialog/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import CardsGallery from '../CardsGallery'
+import Dialog from '../Dialog'
+
+export default React.memo(function HarvestersDialog(props) {
+  return (
+    <Dialog
+      id='harvesters-dialog'
+      dialogRef={instance => (props.dialogRef.current = instance)}
+      close={() => {}}
+      title='Choose the card to copy'
+      image='/assets/images/cards/harvesters_of_souls.png'
+    >
+      {props.cards.length && (
```

Comment:
Do we need this?

Reply:
Without it the CardGallery gets called with no cards and crashes. I could moe this check to HarvestersDialog though

Reply:
I see. We should make the `CardsGallery` component a little more resilient maybe.